crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -102,7 +103,7 @@ pub use wallet::PaymentDetails;
 /// The interval at which the [`lightning::ln::msgs::NodeAnnouncement`] is broadcast.
 ///
 /// According to the LDK team, a value of up to 1 hour should be fine.
-const BROADCAST_NODE_ANNOUNCEMENT_INTERVAL: Duration = Duration::from_secs(3600);
+const BROADCAST_NODE_ANNOUNCEMENT_INTERVAL: Duration = Duration::from_secs(3);
```

Comment:
🔧 I don't think you intend to change that, did you?

Reply:
Good catch, forgot to remove this!